cmd/harmony/main.go

```diff
@@ -803,6 +803,7 @@ func setupConsensusAndNode(hc harmonyconfig.HarmonyConfig, nodeConfig *nodeconfi
 		Uint64("viewID", viewID).
 		Msg("Init Blockchain")
 
+	currentNode.HarmonyConfig.GasPrice = hc.GasPrice
```

Comment:
Why is this needed? 
Consider changing the name of this variable to `GasPriceOracleConfig`

Reply:
Added comment explaining this. Also changed the config from GasPriceConfig to GasPriceOracleConfig.